### PR TITLE
bump manager default limits

### PR DESCRIPTION
**Pull request by stuggi**

### CODE REVIEW

**Comment by stuggi** [comment]
OOMKilled seem in environments, lets bump it

File: config/manager/manager.yaml [hunk 1]
```diff
@@ -34,8 +34,8 @@ spec:
         resources:
           limits:
             cpu: 500m
-            memory: 768Mi
+            memory: 1Gi
```
Comment (olliewalsh) [inline config/manager/manager.yaml:37]: should we make it higher?

Comment (stuggi) [reply config/manager/manager.yaml]: changed to 2Gi

Comment (olliewalsh) [reply config/manager/manager.yaml]: Wonder if we could test this to get a rough idea of the memory requirements? go-git has open issues for using lots of memory....

**Review by olliewalsh — APPROVED** [review]
/lgtm

**Pull request merged**